Patch maxima initialization for sbcl so it uses gmp for arithmetic


--- a/src/init-cl.lisp	2021-05-15 20:20:53.000000000 -0300
+++ b/src/init-cl.lisp	2022-01-04 17:45:03.838273626 -0300
@@ -574,8 +574,13 @@
 			 (delete-file file)))))
 	   *temp-files-list*))
 
+#+sbcl
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (require 'sb-gmp))
+
 (defun cl-user::run ()
   "Run Maxima in its own package."
+  #+sbcl (sb-gmp:install-gmp-funs)
   (in-package :maxima)
   (initialize-runtime-globals)
   (let ((input-stream *standard-input*)
